packages/components/src/components/button/_button.scss

```diff
@@ -444,15 +444,15 @@
   }
 
   .#{$prefix}--btn-set .#{$prefix}--btn.#{$prefix}--btn--disabled {
-    box-shadow: rem(-1px) 0 0 0 $disabled-03;
+    box-shadow: rem(-1px) 0 0 0 $layer-selected-disabled;
```

Comment:
What is this box shadow too? Not sure `$layer-selected-disabled` is the right token for usage.

Reply:
Is it the divider between buttons? It think `$icon-on-color-disabled` fits the usage a little better though still no quite right. We could add `$button-separator-disabled` if we wanted to be super clean about it but I dont think we need too.